irohad/model/queries/get_transactions.hpp

```diff
@@ -34,19 +56,46 @@ namespace iroha {
       std::string account_id{};
 
       /**
-       * Asset identifier
+       * Asset identifiers
        */
-      std::string asset_id{};
+      std::vector<std::string> assets_id{};
+
+      /**
+       * Pager for transactions
+       */
+      Pager pager{};
```

Comment:
Is it a good idea to modify model code (placing pager here)? Imo it's better to make weaker class connection

Reply:
You mean like this
```cpp
struct GetAccountAssetTransactions {
  std::string account_id{};
  std::vector<std::string> assets_id{};
  iroha::hash256_t tx_hash{};
  uint16_t limit{};
  ...
};

struct GetAccountTransactions {
  ...
  iroha::hash256_t tx_hash{};
  uint16_t limit{};
  ...
};
```

or

```cpp
struct GetAccountAssetTransactions {
  ...
  struct Pager { ... };
  Pager pager{};
};
struct GetAccountTransactions {
  ...
  struct Pager { ... };
  Pager pager;
};
```
?

Reply:
@l4l ?

Reply:
Not really understand the first one but definitely not the second one